### ISSUE TITLE
Rights do not flow down

### ISSUE THREAD

**Issue by dbenjamin** [aiaio/mortimer#2]
We have the groups:
SysOp
|___Tech2
       |___Tech1

We have these users:
User1
User2
User3

User1 Write rights to SysOp
User2 Write rights to Tech2
User3 Write rights to Tech1

User2 does not "see" the passwords in the Tech1 group.

**Comment by boltronics** [comment]
Yes, the hierarchy structure inheritance (or lack thereof) is not what you would expect. I wanted a user to have read access to a group, and write access to a specific sub-group, but this does not appear to be possible.
